### PR TITLE
Added my pet info

### PR DESCRIPTION
**Pull request by SoundaryaKoutharapu**
I've added my pet info.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -724,6 +724,48 @@ <h2>Welcome to MyPetSpotlight!</h2>
...
     </div>
     
+
+
+    <div class="card">
+      <img src="./assets/images/lilly.jpg" alt="Lilly - the cat">
+      <div class="content">
+        <div class="name">
+          <strong> Name: </strong>
+          <span> </span>
```
Comment (gabrysia694) [inline index.html:734]: Looks good but please add your pet's name

**Review by gabrysia694 — CHANGES_REQUESTED** [review]

**Comment by SoundaryaKoutharapu** [comment]
Do I have to make another pull request after making changes in my repo.
Can you guide me through this.?

**Comment by SoundaryaKoutharapu** [comment]
Yeah, done

**Review by gabrysia694 — APPROVED** [review]

**Comment by gabrysia694** [comment]
@SoundaryaKoutharapu could you please fetch the latest version of the repository from the main branch, add your pet, and submit the changes in a new pull request, as this pull request has conflicts with the main branch?

**Pull request closed without merging**